renderer.js

```diff
@@ -1249,8 +1340,15 @@ class SDFGRenderer {
         // Toolbar-related fields
         this.menu = null;
         this.toolbar = null;
+        this.panmode_btn = null;
         this.movemode_btn = null;
         this.selectmode_btn = null;
+        this.addmode_btns = null;
+        this.add_type = null;
+        this.addmode_divs = null;
+        this.mode_selected_bg_color = "#CCCCCC";
```

Comment:
Nitpick: The selection color is a lot better the way you do it in context of the VSCode extension - the bright gray is a bit hard to read for standalone SDFV at least, imo :-)